Tests/SwiftFrameTests/Utility/KYBaseTest.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import XCTest
+
+class KYBaseTest: XCTestCase {
```

Comment:
Since we don't use the `KY` prefix anywhere else, maybe enough to call this `BaseTestCase` or something. Just a thought.